optimum/intel/openvino/quantization.py

```diff
@@ -744,6 +784,53 @@ def disable_progress_bar(self, disable: bool = True) -> None:
         else:
             self.model._progress_bar_config["disable"] = disable
 
+    def _prepare_feature_extraction_calibration_data(
+        self,
+        quantization_config: OVQuantizationConfigBase,
+        dataset: "Dataset",
+        seq_len: int = 128,
+    ) -> OVCalibrationDataset:
+        from optimum.intel import OVModelForFeatureExtraction, OVSentenceTransformer
+
+        self.model.compile()
+
+        num_samples = quantization_config.num_samples or 128
+        calibration_data = []
+        try:
+            self.model.request = InferRequestWrapper(
+                self.model.request,
+                calibration_data,
+                inference_result_mock={
```

Comment:
why this empty np arrays are needed?

Reply:
These are for the forward method to be able to post-process the model outputs, for example  https://github.com/huggingface/optimum-intel/blob/main/optimum/intel/openvino/modeling_sentence_transformers.py#L61